### PR TITLE
Add recipes to some more items

### PR DESCRIPTION
**Pull request by ENORMOUZ**

### CODE REVIEW

**Comment by ENORMOUZ** [comment]
Will add bieestery command afteer this pull request is pushed to main

**Comment by ENORMOUZ** [comment]
oops

File: items/VENOMS_TOUCH.json [hunk 1]
```diff
@@ -16,6 +16,17 @@
...
+    "B1": "ENCHANTED_SPIDER_EYE:10",
+    "B2": "",
+    "B3": "SOUL_STRING:1O",
+    "C1": "",
+    "C2": "ENCHANTED_BROWN_MUSHROOM:60",
+    "C3": "SOUL_STRING:1O"
+  },
   "internalname": "VENOMS_TOUCH",
   "crafttext": "",
   "clickcommand": "",
```
Comment (jani270) [inline items/VENOMS_TOUCH.json:32]: you forgot the viewrecipe command here

**Review by jani270 — CHANGES_REQUESTED** [review]

File: items/VENOMS_TOUCH.json (hunk 1 above)
Comment (ENORMOUZ) [reply items/VENOMS_TOUCH.json:32]: oops, fixing.

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**